### PR TITLE
PSP: 1650 Frontend test warning cleanup continued..

### PR DESCRIPTION
**Pull request by jdtoombs**
There are still 3 warnings on MOTI inventory container. Dropped it down though.

Also there are some on EditAgencyPage.test.tsx but I didn't spend more time on this as we may be removing it from the application.

### CODE REVIEW

**Review by Fosol — APPROVED** [review]

File: frontend/src/features/mapSideBar/containers/MotiInventoryContainer.test.tsx [hunk 1]
```diff
@@ -470,58 +488,52 @@ describe('MotiInventoryContainer functionality', () => {
...
-        noButton.click();
-      });
+      fireEvent.click(noButton);
       expect(window.open).not.toHaveBeenCalled();
     });
   });
   describe('populate search form functionality', () => {
     beforeEach(() => {
       history.push('/mapview?sidebar=true&sidebarContext=addBareLand');
+      process.env.REACT_APP_TENANT = 'MOTI';
```
Comment (devinleighsmith) [inline frontend/src/features/mapSideBar/containers/MotiInventoryContainer.test.tsx:536]: hmm, what happens with process.env between tests? do we need to clean this up?

File: frontend/src/features/mapSideBar/containers/MotiInventoryContainer.test.tsx [hunk 2]
```diff
@@ -470,58 +488,52 @@ describe('MotiInventoryContainer functionality', () => {
...
     });
     it('does not display the modal text if there is no duplicate pid', async () => {
       isPidAvailable.mockResolvedValue({ available: true });
       const { container, getByTestId } = renderContainer({});
-      await fillInput(container, 'searchPid', '123-456-789');
-      act(() => {
-        fireEvent.click(getByTestId('pid-search-button'));
-      });
+      const searchPid = container.querySelector('input[name="searchPid"]');
+      await waitFor(() => fireEvent.change(searchPid!, { target: { value: '123-456-789' } }));
```
Comment (devinleighsmith) [inline frontend/src/features/mapSideBar/containers/MotiInventoryContainer.test.tsx:507]: this is interesting did we add this as a result of a warning? waitFors are normally used to essentially wait for an async page change that is required by some logic. However, there shouldn't be a page change required for changing the searchPid field is there? Does this test work without that waitFor?

File: frontend/src/features/mapSideBar/containers/MotiInventoryContainer.tsx [hunk 3]
```diff
@@ -193,7 +193,7 @@ export const MotiInventoryContainer: React.FunctionComponent = () => {
         toast.error(
           'Property search failed. Please check your search criteria and try again. If this error persists, contact the Help Desk.',
         );
-        console.debug(error);
+        // console.debug(error);
```
Comment (devinleighsmith) [inline frontend/src/features/mapSideBar/containers/MotiInventoryContainer.tsx:196]: If this was as a result of a warning, just delete it. I don't see a reason to add it back.

**Review by devinleighsmith — APPROVED** [review]

**Comment by devinleighsmith** [comment]
@jdtoombs can you recreate this PR on a new branch when you get a minute? We aren't going to fix the underlying issue preventing the checks from running.

**Comment by jdtoombs** [comment]
Trouble with GitHub actions, need to close

**Comment by asanchezr** [comment]
@jdtoombs yeah, that is a side effect of this PR being sent from your **dev** branch instead of a feature branch. I need to improve the versioning action but have no time ATM for that as other higher priority issues need to be addressed first.

A PR from a feature branch (ie "psp-1650-test-warnings") should be ok. The version action only triggers on pushes to `dev`

**Pull request closed without merging**